app/client/src/utils/utils.ts

```diff
@@ -62,10 +60,48 @@ function getExtensionFromPath(primary: string, backup: string = '') {
   return extension;
 }
 
-require('@gouch/to-title-case');
-
 function titleCase(string: string) {
-  return string.toLowerCase().toTitleCase();
+  const smallWords =
+    /^(a|an|and|as|at|but|by|en|for|if|in|nor|of|on|or|per|the|to|v.?|vs.?|via)$/i;
+  const alphanumericPattern = /([A-Za-z0-9\u00C0-\u00FF])/;
+  const wordSeparators = /([ :–—-])/;
+
+  return string
+    .toLowerCase()
+    .split(wordSeparators)
+    .map(function (current, index, array) {
+      if (
+        // check for small words
+        current.search(smallWords) > -1 &&
+        // skip first and last word
+        index !== 0 &&
+        index !== array.length - 1 &&
+        // ignore title end and subtitle start
+        array[index - 3] !== ':' &&
+        array[index + 1] !== ':' &&
+        // ignore small words that start a hyphenated phrase
+        (array[index + 1] !== '-' ||
+          (array[index - 1] === '-' && array[index + 1] === '-'))
+      ) {
+        return current.toLowerCase();
+      }
+
+      // ignore intentional capitalization
+      if (current.substring(1).search(/[A-Z]|\../) > -1) {
```

Comment:
This code was pulled from the `@gouch/to-title-case` dependency that I removed. We may want to simplify this code at some point, since some if it likely doesn't apply to what we use it for.

It looks like they added that code to ignore words that are already capitalized. So if you give it something like `first sEcOnd third pH` then the result would be `First sEcOnd Third pH`, where `sEcOnd` and `pH` are ignored because those words already have capital letters in them.

Reply:
Ah. Yeah I'm not entirely sure what their intention was since it was pulled from that old dependency.